api/all/docs/api_diff_1.1.0_vs_1.0.0.txt

```diff
@@ -0,0 +1,2 @@
+Comparing source compatibility of  against 
```

Comment:
I think it'd be better to aggregate all these into top level, `docs/${newVersion}/${archivesBaseName}` to get an idea of all the changes in a version more easily.

Reply:
yeah, at one point I had this implemented and decided it would be better to keep the diffs close to the source, but I don't feel super strongly about that.

Also, I did want to make sure that we saw diffs vs. the latest release. what directory would that end up in, do you think?  Or, would we just put the concrete ones up at the top level for the historical records, and put the incremental snapshot changes inside the modules?

Reply:
I think I'd still be find with diff vs current (`docs/current/`) to be aggregated, and we get to see all the new API in the next version very easily.

Reply:
Both of my suggestions should have been `docs/apidiffs` by the way :)

Reply:
cool. I'll update this to put everything under docs/apidiffs:

`docs/apidiffs/current/<module>.txt`
`docs/apidiffs/v1.1.0_v1.0.0/<module>.txt`

Reply:
ok, moved and regenerated. please take a look.